code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -43,7 +43,7 @@
 	var/list/msg = list("<span class='info'>*---------*\nThis is ")
 
 	var/datum/gender/T = gender_datums[gender]
-	if(skipjumpsuit && skipface) //big suits/masks/helmets make it hard to tell their gender
+	if(skipjumpsuit && skipface || IS_SKRELL) //big suits/masks/helmets make it hard to tell their gender

```

Comment:
```suggestion
	if((skipjumpsuit && skipface) || IS_SKRELL) //big suits/masks/helmets make it hard to tell their gender
```

I'm also uncertain if this is even needed?

Reply:
oh no. this is not how you do it. aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.